### ISSUE TITLE
How to test webrtc experiments

### ISSUE THREAD

**Issue by Globik** [xhs/webrtc-experiments#1]
Excuse me, I look at test_sctp_dtls_ice.c. What is it and how to test it? One need two consoles? For server and client? Is it testing WebRTC datachannel over libnice connectivity plus dtls, right?

**Comment by xhs** [comment]
Hi @Globik , these are some PoC prototypes I made during the development of librtcdc (https://github.com/xhs/librtcdc). And yes, they are testing WebRTC DataChannel over SCTP over DTLS over libnice connectivity. Usually you will need to open two terminals to test them.

**Comment by Globik** [comment]
Yes, i have just compiled this test sample. Opened two terminales, for server, for client. My issue that i am coming always to the step of "continues handshaking". It looks like function SSL_is_init_finished(dtls->ssl) always is not equal to 1.
Anyway thank you for your nice test samples they give a good idea where to start with that all server side webrtc stack, with all that protocols. And many thanks for librtcdc.so It can be used also on pure C or it is only for cython binding to python experimentally?
